local-cli/generator/index.js

```diff
@@ -27,6 +26,11 @@ module.exports = yeoman.generators.NamedBase.extend({
       type: Boolean,
       defaults: false
     });
+    this.option('skip-jest', {
+      desc: 'Skip installing jest',
```

Comment:
"Jest" (capitalized) in writing please. `jest` is the name of the cli tool for "Jest".